### ISSUE TITLE
Unable to release the memory occupied by NativeWindow 

### ISSUE THREAD

**Issue by xiejiang2014** [opentk/opentk#1304]
### Description

NativeWindow.Dispose();    Cannot release memory 

### Repro steps

```
using System;
using OpenTK.Windowing.Common;
using OpenTK.Windowing.Desktop;

namespace ConsoleApp1
{
    class Program
    {
        static void Main(string[] args)
        {
            var nws = NativeWindowSettings.Default;
            nws.StartFocused    = false;
            nws.StartVisible    = false;
            nws.NumberOfSamples = 0;

            // if we ask GLFW for 1.0, we should get the highest level context available with full compat.
            nws.APIVersion = new Version(2, 1);
            nws.Flags      = ContextFlags.Offscreen | ContextFlags.Default;

            // we have to ask for any compat in this case.
            nws.Profile      = ContextProfile.Any;
            nws.WindowBorder = WindowBorder.Hidden;
            nws.WindowState  = OpenTK.Windowing.Common.WindowState.Minimized;

            for (var i = 0; i < 100; i++)
            {
                var nativeWindow = new NativeWindow(nws);
                nativeWindow.Dispose();
            }

            GC.Collect();

            Console.WriteLine("Please check the memory.");
            Console.ReadLine();
        }
    }
}
```

### Expected behavior

Release all memory occupied by NativeWindow. 

### Actual behavior

Unable to release all the memory occupied by NativeWindow, the process continues to occupy about 485 MB of memory

### Related information 

Windows 10    [10.0.19041.985]
NuGet 4.6.4
net5.0
Intel UHD 630   Driver: 27.20.100.8935

**Comment by deccer** [comment]
After a short glimpse into `NativeWindow.Dispose` which doesn't seem to be doing anything at all :) could explain why

**Comment by xiejiang2014** [comment]
> After a short glimpse into `NativeWindow.Dispose` which doesn't seem to be doing anything at all :) could explain why.

Thank you.

The above code is just to demonstrate that even if I don't do any other operations, the memory occupied by NativeWindow cannot be released.

The actual situation is that I use multiple GLWpfControl (from nuget OpenTK.GLWpfControl 4.1.0) in the WPF program.I noticed that as the GLWpfControl instance is loaded and unloaded, the memory of the program continues to increase, and finally I found that the problem may come from NativeWindow.

> I also assume by `Nuget 4.6.4` you mean `OpenTK 4.6.4`. Correct?

Yes, OpenTK 4.6.4 from nuget, my mistake ,sorry.
